java-components/domainproxy/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM quay.io/redhat-appstudio/buildah:v1.31.0@sha256:34f12c7b72ec2c28f1ded0c494b428df4791c909f1f174dd21b8ed6a57cf5ddb
+RUN dnf install -y iproute
+COPY client/target/domainproxy-client-1.0.0-SNAPSHOT-runner /app/domainproxy-client-runner
+COPY server/target/domainproxy-server-1.0.0-SNAPSHOT-runner /app/domainproxy-server-runner
```

Comment:
Is this the primary Dockerfile? i.e. are java-components/domainproxy/client/src/main/docker and java-components/domainproxy/servicer/src/main/docker required?

Reply:
Unsure about this. @stuartwdouglas Are these docker dirs needed? I needed to package the native artifacts for use in a Konflux pipeline.

Reply:
The other ones are not required, they were automatically generated when the project was created.